doc/Security.xml

```diff
@@ -947,6 +960,61 @@ Added certificate-based client authentication</revremark>
         </orderedlist>
       </section>
     </section>
+    <section>
+      <title>JWT-based client authorization</title>
+      <para>Unlike with HTTP Digest and WS-UsernameToken authentication, this specification defines how to associate clients with the different User Levels as defined in the ONVIF Core Specification. Devices are not expected to have the user credentials stored in their memory, instead they will verify that the user has been correctly authenticated by an external service based on Open ID Connect and authorize accessing different functions, based on the user level presented in  the supplied JWT embedded in the binary security token.</para>
+
+      <para>The content of the binary security token is the base64-encoded JWT according to [RFC 7519].</para>
+      <para>The JOSE header shall declare that the encoded object is a JWT, and the JWT is a JWS that is hashed using the HMAC SHA-256. Additionally, it must include the kid claim as defined by [RFC 7515], which indicates the ID associated to the key used to sign the JWT. If the absolute path of the JWK Set URL different from  https://iss/.well-known/jwks.json, the header may include an optional jku claim.</para>
+      <para>The JWT shall include the following standard claims:</para>
+      <para>
+        <itemizedlist>
+          <listitem>
+            <para><emphasis>iss</emphasis>: Issuer, the FQDN of the server that generated the JWT.</para>
```

Comment:
Based on https://www.rfc-editor.org/rfc/rfc7519#section-4.1.1

> [4.1.1](https://www.rfc-editor.org/rfc/rfc7519#section-4.1.1).  "iss" (Issuer) Claim
> 
>    The "iss" (issuer) claim identifies the principal that issued the
>    JWT.  The processing of this claim is generally application specific.
>    The "iss" value is a case-sensitive string containing a StringOrURI
>    value.  Use of this claim is OPTIONAL.

So, since it is a StringOrURI, I understood it must be a FQDN, but to be pedantic it could be a URI as well, so it could be either `my.server.com` or `https://my.server.com`.

The usage of the iss claim is bound to the open point about how the device can know if i has to trust a issuer or not.

Reply:
> Based on https://www.rfc-editor.org/rfc/rfc7519#section-4.1.1
> 
> > [4.1.1](https://www.rfc-editor.org/rfc/rfc7519#section-4.1.1).  "iss" (Issuer) Claim
> > The "iss" (issuer) claim identifies the principal that issued the
> > JWT.  The processing of this claim is generally application specific.
> > The "iss" value is a case-sensitive string containing a StringOrURI
> > value.  Use of this claim is OPTIONAL.
> 
> So, since it is a StringOrURI, I understood it must be a FQDN, but to be pedantic it could be a URI as well, so it could be either `my.server.com` or `https://my.server.com`.
> 
> The usage of the iss claim is bound to the open point about how the device can know if i has to trust a issuer or not.

Thanks for the response to my question I do not want it to deflect from the suggested update on the line text.